api/resolvers/wallet.js

```diff
@@ -65,7 +67,12 @@ function injectResolvers (resolvers) {
         wallet,
         testCreateInvoice:
           walletDef.testCreateInvoice && validateLightning && canReceive({ def: walletDef, config: data })
-            ? (data) => walletDef.testCreateInvoice(data, { logger })
+            ? (data) => withTimeout(
+                walletDef.testCreateInvoice(data, {
+                  logger,
+                  signal: timeoutSignal(WALLET_CREATE_INVOICE_TIMEOUT_MS)
+                }),
+                WALLET_CREATE_INVOICE_TIMEOUT_MS)
```

Comment:
Mhhh, if the wallet can use signals, then it's not clear if `withTimeout` or the signal will throw first :thinking: 

Preferably, we want the signal to throw first so we get `FetchTimeoutError` as the error which includes additional information.

I tested this with LNbits and setting the timeout to 10 and the signal always won but not sure if we can rely on that. I'll add 100ms to `withTimeout` to be sure but without affecting the timeout message itself.

Reply:
done in db5a174e